dashboard/app/bug_impact_score.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import "regexp"
+
+// impactOrder represent an ordering of bug impact severity. The earlier
+// entries are considered more severe.
+// The regexps are matched in order, and the first match determines the
+// impact category. A higher index in this slice means lower impact.
+// The last regexp ".*" is a catch-all for unknown bugs.
+var impactOrder = []string{
+	// Memory safety.
+	`^KASAN:.*Write`,
+	`^KASAN:.*Read`,
```

Comment:
What about invalid-free/double-free? These look as bad as writes to me. Maybe worse.